packages/ens-controller/package.json

```diff
@@ -29,8 +29,12 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@ethersproject/providers": "^5.7.0",
     "@metamask/base-controller": "workspace:^",
-    "@metamask/controller-utils": "workspace:^"
+    "@metamask/controller-utils": "workspace:^",
+    "@metamask/utils": "^3.3.1",
```

Comment:
Is this still being used?

Reply:
@Gudahtt I used it to import the logger used to import the logger
import { createProjectLogger } from '@metamask/utils';

Reply:
Oh sorry, I missed that, thanks